app/cli.py

```diff
@@ -287,28 +286,28 @@ def main(
             set_and_verify_aws_credentials(region_name=_region)
 
     if aws_ipi_clusters:
-        clusters = generate_cluster_dirs_path(
+        aws_ipi_clusters = generate_cluster_dirs_path(
             clusters=aws_ipi_clusters, base_directory=clusters_install_data_directory
         )
 
-        clusters = download_openshift_install_binary(
-            clusters=clusters, registry_config_file=registry_config_file
+        aws_ipi_clusters = download_openshift_install_binary(
+            clusters=aws_ipi_clusters, registry_config_file=registry_config_file
         )
         if create:
-            clusters = create_install_config_file(
-                clusters=cluster,
+            aws_ipi_clusters = create_install_config_file(
+                clusters=aws_ipi_clusters,
```

Comment:
@myakove Another solution would be to change to `clusters=clusters` but I think the existing change is more clear, wdyt?